cfme/infrastructure/cluster.py

```diff
@@ -132,6 +133,43 @@ def is_displayed(self):
             super(TimelinesView, self).is_displayed)
 
 
+class ClusterCollection(Navigatable):
```

Comment:
**REQURIED**
`ClusterCollection` should have `create()` and `instantiate()` methods. Look at Automate code as a reference.

Reply:
Yes, instantiate is a must, create depends :)